### PR TITLE
Week 1

### PR DESCRIPTION
**Pull request by sriratnavallidhulipala**
- **Please review the PR  & provide the feedback** 

[Catty Board](https://music-cattyboard.netlify.app/)

### CODE REVIEW

File: Week-1/Assignments/index.css [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+
+body {
+    font-family: 'Lato', sans-serif;
+}
+
+/* styles for layout */
+nav {
+    background-color: rgba(231, 220, 220, 0.899);
+    width: 3%;
+    height: 85vh;
```
Comment (sethiaPiyush) [inline Week-1/Assignments/index.css]: we shouldn't be writing such specific values to css properties as it leads to unresponsiveness

File: Week-1/Assignments/index.css [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+    list-style: none;
+}
+
+nav ul :first-child i {
+    margin-top: 3vh;
+}
+
+nav i {
+    color: black;
+    margin-bottom: 5.5vh;
```
Comment (sethiaPiyush) [inline Week-1/Assignments/index.css]: same goes for this one

**Review by sethiaPiyush — APPROVED** [review]
the architecture of the files and code could be done better. WIll discuss the same in next session

File: Week-1/Assignments/index.css (hunk 1 above)
Comment (sriratnavallidhulipala) [reply Week-1/Assignments/index.css]: okay will do it

File: Week-1/Assignments/index.css (hunk 2 above)
Comment (sriratnavallidhulipala) [reply Week-1/Assignments/index.css]: will do it

**Review by sethiaPiyush — APPROVED** [review]

**Pull request merged**